src/app/participant-list/participant-list.component.html

```diff
@@ -355,20 +355,26 @@ <h3 class="Color--primary Line--Break">{{additionalMessage}}</h3>
                           </ng-container>
                           <ng-container
                             *ngIf="questionAnswer.questionType === 'DATE' && questionAnswer.dateFields != null">
-                            <ng-container *ngIf="questionAnswer.dateFields.day != null && questionAnswer.dateFields.day !== '' &&
-questionAnswer.dateFields.month != null && questionAnswer.dateFields.month !== ''">
-                              {{questionAnswer.dateFields.month}}/{{questionAnswer.dateFields.day}}/{{questionAnswer.dateFields.year}}
+                            <ng-container *ngIf="pt.data.getMultipleDatesForActivity(activityData, col.participantColumn.name) as answers">
```

Comment:
this change is to display multiple date activities like diagnosis date correctly